tests/OK/inductive_type.lp

```diff
@@ -0,0 +1,16 @@
+inductive nat : TYPE ≔
+ | z    : nat
+ | succ : nat → nat
+
+assert z : nat
+assert succ z : nat
+assert succ (succ (succ z)) : nat
+
```

Comment:
Thanks to add a test to check that the induction principle is generated with the correct type. Idem for list below.

Reply:
Les symboles "nat_ind" et "list_ind" ne sont pas connus.

Reply:
They should be...